src/styles/_catalog.scss

```diff
@@ -1,22 +1,47 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  --column-count: 1;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 48px 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
 }
+
+@media (max-width: 488px) {
+  .catalog {
+    --column-count: 1;
+  }
+}
```

Comment:
You don't need this media query you have already set 1 as default value on 2 line